src/services/data/DataService.tsx

```diff
@@ -7,11 +8,16 @@ export class DataService {
   static postMultipartRequest(data: any, fullURI?: string): Promise<any> {
```

Comment:
Open to opinions/arguments otherwise, but thought I'd comment. I like to use the following pattern for this type of class instead of using static methods:

```js
class DataService {
  baseURI: string;
  constructor(baseURI: string) {
    this.baseURI = baseURI;
  }
  // ...methods
}

export const dataService = new DataService(process.env.REACT_APP_MCP_DATA_MAPPING_UTIL);
```

kinda singleton pattern-esque

Reply:
I'm not sure how this would affect testing with jest spies though

Reply:
Done.